screens/History/HistoryScreen.tsx

```diff
@@ -5,11 +5,10 @@ import {useTranslation} from 'react-i18next';
 import {Centered, Column, Text} from '../../components/ui';
 import {useHistoryTab} from './HistoryScreenController';
 import {ActivityLogText} from '../../components/ActivityLogText';
-import {MainRouteProps} from '../../routes/main';
 import {Theme} from '../../components/ui/styleUtils';
 import {BannerNotificationContainer} from '../../components/BannerNotificationContainer';
 
-export const HistoryScreen: React.FC<MainRouteProps> = () => {
+export const HistoryScreen: React.FC = () => {
```

Comment:
Doubt: If there were no **MainRouteProps**, in successful share modal the history icon navigation might be get affected . Did you check that?

Reply:
MainRouteProps are not using in this component, we are navigating through bottomTabRoutes. There is no use of it so removed.